Bonsai.Editor/DocumentationHelper.cs

```diff
@@ -61,11 +61,23 @@ static async Task<Uri> GetDocumentationAsync(string baseUrl, string uid)
             using var response = await request.GetResponseAsync();
             var stream = response.GetResponseStream();
             using var reader = new StreamReader(stream);
+
+            if (reader.ReadLine().Trim() != "### YamlMime:XRefMap")
+                throw new DocumentationException("The documentation server did not respond with a cross-reference map.");
+
             var deserializer = new DeserializerBuilder()
                 .WithNamingConvention(CamelCaseNamingConvention.Instance)
                 .IgnoreUnmatchedProperties()
                 .Build();
-            var xrefmap = deserializer.Deserialize<XRefMap>(reader);
+            XRefMap xrefmap;
+            try
+            { xrefmap = deserializer.Deserialize<XRefMap>(reader); }
+            catch (YamlException ex)
+            { throw new DocumentationException("The cross-reference map returned by the documentation server is malformed.", ex); }
```

Comment:
```suggestion
            try { xrefmap = deserializer.Deserialize<XRefMap>(reader); }
            catch (YamlException ex)
            {
                throw new DocumentationException("The cross-reference map returned by the documentation server is malformed.", ex);
            }
```

For same style reasons as above.